src/JMS/Serializer/Annotation/XmlCollection.php

```diff
@@ -34,4 +34,9 @@
      * @var string
      */
     public $namespace;
+
+    /**
+     * @var boolean
+     */
+    public $skip_when_empty = true;
```

Comment:
Do we use this casing for other annotations? I think we already use camelCasing, no?